mitodl/open-discussions
# Set membership_is_managed to False when creating channels from app

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #1392

#### What's this PR do?
- Sets `membership_is_managed` to False when creating new channels from within open-discussions.  Channels created from micromasters should still be set to True.

#### How should this be manually tested?
- Go to `/manage/c/new` as a staff user and create a new channel.  Go to the moderators page of that channel, you should be able to add moderators.
- From micromasters, log in as `staff@example.com` and create a new channel.  Go to the moderators page of that channel, you should not be able to add moderators, it should display the message `Membership is managed via MicroMasters`
